content/cybersecurity/concepts/cryptography/cryptography.md

```diff
@@ -0,0 +1,13 @@
+---
+Title: 'Cryptography' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'A method of protecting information and communications through the use of codes that allow only the sender and receiver of a message to view its contents.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
+Subjects: # Please only use Subjects in the subjects.md file (https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). If that list feels insufficient, feel free to create a new Subject and add it to subjects.md in your PR!
+  - 'Cybersecurity'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'Cybersecurity'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Cryptography** provides confidentiality, integrity, authentication for sensitive information while it is stored (at rest), traveling across a network (in transit), and existing in memory (in use). Two main types of cryptosystems enforce confidentiality: symmetric and asymmetric cryptosystems.
```

Comment:
Can we have more than a Glossary entry here? This is a major subject and deserves a couple of paragraphs at least. Uses, history, modern ciphers, etc.